### PR TITLE
Master: update deps

### PR DESCRIPTION
**Pull request by ap-justin**
Ticket(s):
-

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
     "react-redux": "^8.0.5",
     "react-router-dom": "^6.2.0",
     "react-scripts": "5.0.1",
-    "type-fest": "^3.9.0",
+    "type-fest": "4.5.0",
```
Comment (misicnenad) [inline package.json:82]: Won't this break due to versions starting from 4.0.0 requiring TypeScript 5.1?
[See release notes for v4.0.0](https://github.com/sindresorhus/type-fest/releases/tag/v4.0.0)

Comment (ap-justin) [reply package.json]: reverted ser https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2444/commits/2e2139d3cfa9e09997a5d591366ff197378bfbd8

**Review by misicnenad — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (misicnenad) [reply package.json]: If you see this comment before the PR is merged, just saw that the last v3 of the package is [v3.13.1](https://github.com/sindresorhus/type-fest/releases/tag/v3.13.1), we can upgrade to that

**Review by misicnenad — CHANGES_REQUESTED** [review]
Requesting changes just to raise this to your attention - the last v3 of the `type-fest` package is [v3.13.1](https://github.com/sindresorhus/type-fest/releases/tag/v3.13.1), so we can upgrade to that.
Otherwise, the PR is good to go and can be merged :+1:

File: package.json (hunk 1 above)
Comment (ap-justin) [reply package.json]: `v3.13.1` also depends on typescript `v5`, so I just reverted to this version that works.  This shoudn't be a blocker for this PR

**Review by misicnenad — APPROVED** [review]

**Pull request merged**